core/descriptors.jvm/src/org/jetbrains/kotlin/load/java/JvmAbi.java

```diff
@@ -49,18 +49,12 @@
     public static final ClassId REFLECTION_FACTORY_IMPL = ClassId.topLevel(new FqName("kotlin.reflect.jvm.internal.ReflectionFactoryImpl"));
 
     /*
-        Line number is used when inlining function arguments to @InlineOnly functions.
-        All line information is stripped from the inlined implementation of such functions,
-        so if a lambda is on the same line with the call, the debugger can't stop on both 'line' and 'lambda' breakpoints.
-
-        Example:
-            > require(foo) { "foo is false" }
+        Synthetic line number (see `SourceMapper.mapSyntheticLineNumber`) used when inlining function arguments
+        to @InlineOnly functions. All line information is stripped from the inlined implementation of such
+        functions, so if a lambda is on the same line with the call, the debugger can't stop on both 'line'
+        and 'lambda' breakpoints (KT-23064).
      */
-    public static final int LOCAL_VARIABLE_INLINE_ARGUMENT_SYNTHETIC_LINE_NUMBER = 65100;
-
-    public static final List<Integer> SYNTHETIC_MARKER_LINE_NUMBERS = CollectionsKt.listOf(
-            LOCAL_VARIABLE_INLINE_ARGUMENT_SYNTHETIC_LINE_NUMBER
-    );
+    public static final int LOCAL_VARIABLE_INLINE_ARGUMENT_SYNTHETIC_LINE_NUMBER = 1;
```

Comment:
@pyos I would prefer to move out this constant from JvmAbi.
Client (debugger) could determine syntheticness by  ` kotlin/jvm/internal/FakeKt ` source

Reply:
`inlineCodegenUtils.kt` looks as better place

Reply:
But if later debugger needs to determine *which* line number it is (like for the step-over-suspend case mentioned above), wouldn't the constant need to be in JvmAbi?

Reply:
Not necessery

Reply:
Now coruines related staff implemented via additional local variables. If it would be neccessery it could be rethinked later

Reply:
Ok, moved the constant to InlineOnlySmapSkipper.